data/mods/NPC_Traits/modinfo.json

```diff
@@ -1,11 +0,0 @@
-[
-  {
-    "type": "MOD_INFO",
-    "ident": "npc_traits",
-    "name": "NPC traits",
-    "authors": [ "Spencer Michaels" ],
-    "description": "Add an NPC class that spawns with starting traits, like the player does.",
-    "category": "misc_additions",
-    "dependencies": [ "dda" ]
```

Comment:
This needs to be marked as obsoleted instead of removing , don't break saves.

Reply:
you add one line `"obsolete": true` in the modinfo.json.

See https://github.com/CleverRaven/Cataclysm-DDA/pull/19398/commits/b93567b9093cda30f2dbbad2ecca88289ac67cc0

Reply:
Thanks!